src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -492,33 +508,33 @@ private static void ParseArgumentsIntoList(string arguments, List<string> result
                 // it contains spaces.
                 if (c == '"')
                 {
-                    inQuotes = !inQuotes;
+                    if (inQuotes && i < arguments.Length - 1 && arguments[i + 1] == '"')
+                    {
+                        currentArgument.Append('"');
```

Comment:
This is worth a comment as it's implementing a rule that's not documented in that parsing link.

Something like a "Two consecutive double quotes inside an inQuotes region causes a literal double quote to be emitted and leaves the parser in the inQuotes region. For compatibility with post-2008 CRT's and CLR 4.0"